boat-scaffold/src/test/java/com/backbase/oss/codegen/java/BoatSpringCodeGenTests.java

```diff
@@ -74,4 +88,37 @@ void addServletRequestTestFromOperation(){
         assertEquals("httpServletRequest", co.allParams.get(0).paramName);
         assertTrue(Arrays.stream(co.allParams.get(0).getClass().getDeclaredFields()).anyMatch(f -> "isHttpServletRequest".equals(f.getName())));
     }
+
+    @Test
+    void multipartWithFileAndObject() throws IOException {
+        var codegen = new BoatSpringCodeGen();
+        var input = new File("src/test/resources/boat-spring/multipart.yaml");
+        codegen.setLibrary("spring-boot");
+        codegen.setInterfaceOnly(true);
+        codegen.setOutputDir(TEST_OUTPUT + "/multipart");
+        codegen.setInputSpec(input.getAbsolutePath());
+
+        var openApiInput = new OpenAPIParser().readLocation(input.getAbsolutePath(), null, new ParseOptions()).getOpenAPI();
+        var clientOptInput = new ClientOptInput();
+        clientOptInput.config(codegen);
+        clientOptInput.openAPI(openApiInput);
+
+        var files = new DefaultGenerator().opts(clientOptInput).generate();
```

Comment:
Please only use `var` in case the type is explicit in the context, i.e.
**GOOD** (you know that codegen is of type BoatSpringCodeGen)
```
var codegen = new BoatSpringCodeGen();
```
**BAD** (you don't know files type)
```
var files = new DefaultGenerator().opts(clientOptInput).generate();
```

Reply:
Made the ones more explicit where the context is not super clear